ui/pages/Address.tsx

```diff
@@ -233,7 +218,7 @@ const AddressPageContent = () => {
       <AddressDetails addressQuery={ addressQuery } scrollRef={ tabsScrollRef }/>
       { /* should stay before tabs to scroll up with pagination */ }
       <Box ref={ tabsScrollRef }></Box>
-      { (addressQuery.isPlaceholderData || addressTabsCountersQuery.isPlaceholderData) ? <TabsSkeleton tabs={ tabs }/> : content }
```

Comment:
You can keep the placeholder condition for the address counters: 
`|| addressCountersQuery.isPlaceholderData`